fennel/client/client.py

```diff
@@ -759,6 +759,34 @@ def inspect(
 
         return self._parse_dataframe(result, output_dtypes)
 
+    # ----------------------- Secret API's -----------------------------------
+
+    def get_secret(self, secret_name: str) -> Optional[str]:
```

Comment:
I set a set_secrets in the mock client but not here? lets keep parity and add it to both ?

Reply:
yup, reomved set_secret in mock client.